### PR TITLE
Create dcsa_event_domain.yaml

### PR DESCRIPTION
**Pull request by HenrikHL**
Initial commit

### CODE REVIEW

File: domain/dcsa_event_domain.yaml [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+                - GTIN (Gate in)
+                - GTOT (Gate out)
+                - STUF (Stuffing)
+                - STRP (Stripping)
+        - type: object
+          properties:
+            eventClassifierCode:
+              description: |
+                Code for the event classifier can be:
+                - PLN (Planned)
```
Comment (HenrikHL) [inline domain/dcsa_event_domain.yaml:339]: ´´´suggestion
                - EST (Estimated)
```

File: domain/dcsa_event_domain.yaml [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+                - ACT (Actual)
+              enum:
+                - ACT
```
Comment (HenrikHL) [inline domain/dcsa_event_domain.yaml:342]:
```suggestion
                - EST
```

File: domain/dcsa_event_domain.yaml [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+                - PLN
```
Comment (HenrikHL) [inline domain/dcsa_event_domain.yaml:343]:
```suggestion
                - EST
```

File: domain/dcsa_event_domain.yaml (hunk 1 above)
Comment (HenrikHL) [inline domain/dcsa_event_domain.yaml:339]:
```suggestion
                - EST (Estimated)
```

**Review by holmHin — APPROVED** [review]

**Pull request merged**